index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="portfolio.css">
+    <title>Nsanzimfura portfolio</title>
+
+</head>
+<body>
+    <header>
+        <nav class="top-navigation-bar">
+            <h1 class="header-welcome">Welcome</h1>
+        </nav>
+        <img class="menu-icon" src="./assets/header-files/icon-left-side-menu.svg" alt="hamburger-icon" />
+    </header>
```

Comment:
This looks great 👌but it's highly recommended to follow the exact Figma design, so in this case please make sure to remove the extra space from the top.

![image](https://user-images.githubusercontent.com/34779928/198398144-c502a851-0435-46e6-b6ef-5f921b55a0b4.png)

Reply:
- [ ] Please correct the white space on the top of the element so it follows the exact Figma design 💯